flask_openapi3/scaffold.py

```diff
@@ -48,7 +49,7 @@ def _do_decorator(
             operation_id: Optional[str] = None,
             extra_form: Optional[ExtraRequestBody] = None,
             extra_body: Optional[ExtraRequestBody] = None,
-            responses: Optional[Dict[str, Union[Type[BaseModel], Dict[Any, Any], None]]] = None,
+            responses: Optional[ResponseDict] = None,
```

Comment:
Why not use inheritance for GET, PUT, DELETE, etc and only overwrite the http method?
Every time there is a change, you need to change like 5-6 classes. It's inconvenient for you.

Reply:
I think this code is more straightforward and easier to understand, and if you have a better way, I suggest reopening a PR to discuss.

Reply:
Understood, I'm not sure if my way is technically better. I'll propose one and show what I mean later. Maybe you'll like it.